ocaml/tests/test_helpers.ml

```diff
@@ -284,10 +284,40 @@ module CIDRCheckers = Generic.MakeStateless (struct
     ]
   end)
 
+module AreOnSameSubnetTests = Generic.MakeStateless (struct
+  module Io = struct
+    type input_t = string * string list
```

Comment:
Minor style issue: I don't think types need to be marked with a `_t` suffix. They live in their own name space and can't collide with value names. It is a customary to use `t` for the central type of a module, though (like in `Stack.t`).

Reply:
i think adding the `_t` helps with readability, since we use snake case for both types and variable names

Reply:
`input_t` and `output_t` seem to be a convention in the tests of xapi, I found 128 uses:
```
$ rg 'type [^ ]+put_t' -c | cut -d ":" -f 2 | jq -s 'add'
128
```

Reply:
to would be easy to change (you just need to change the signature of the `MakeStateless` functor, and the compiler will tell you what needs fixing), but i can't see it being worthwhile